packages/blockchain-wallet-v4-frontend/src/services/LockboxService/utils.js

```diff
@@ -83,16 +83,16 @@ const createDeviceSocket = (transport, url) => {
     }
 
     ws.onopen = () => {
-      //console.info('OPENED', { url })
+      console.info('OPENED', { url })
     }
 
     ws.onerror = e => {
-      //console.info('ERROR', { message: e.message, stack: e.stack })
+      console.info('ERROR', { message: e.message, stack: e.stack })
       o.error(e.message, { url })
     }
 
     ws.onclose = () => {
-      //console.info('CLOSE')
+      console.info('CLOSE')
```

Comment:
are these left uncommented on purpose?

Reply:
We had them uncommented for a very long time for debugging purposes in higher envs. I just re-enabled them as we have a revamp of a few Lockbox features coming up.  

Will definitely disable again though in the future.

Reply:
yeah seen them working when tested lockbox